app/src/main/res/values-bn-rBD/strings.xml

```diff
@@ -193,8 +193,8 @@
   <!-- PIN entry -->
   <string name="pinentry_forgotpin">পিন ভুলে গেছেন?</string>
   <string name="pinentry_error_incorrect_pin_on_first_attempt">পিনটি আসল পিনটির সাথে মেলে না</string>
-  <string name="pinentry_error_incorrect_pin_on_subsequent_attempts" tools:ignore="PluralsCandidate">ভুল PIN, আরো %d বার চেষ্টা করতে পারবেন</string>
-  <string name="pinentry_error_incorrect_attempts_limit_reached" tools:ignore="PluralsCandidate">আপনি %d বার ভুল প্রচেষ্টা করেছেন</string>
+  <string name="pinentry_error_incorrect_pin_on_subsequent_attempts" tools:ignore="PluralsCandidate">ভুল PIN, আরো %s বার চেষ্টা করতে পারবেন</string>
```

Comment:
I think we can remove the `tools:ignore="PluralsCandidate"` now since that will only be checked for strings which contain an integer placeholder.

Reply:
As discussed, I am not making this change. This will be handled by Transifex. If not, we will make this change later.